package.json

```diff
@@ -28,8 +28,10 @@
     "babel-loader": "^6.4.1",
     "babel-preset-es2015": "^6.24.1",
     "babel-preset-stage-1": "^6.24.1",
+    "blendid": "^4.2.0",
```

Comment:
should blendid be installed on blendid?

Reply:
I didn't think you could do this. What does this solve?

Reply:
mostly so you could run the blendid node package within the repo but then the node package will be installed on the node package which will create an infinity mirror, maybe. ill remove this because this was only really needed so i could test my work within the repo.

Reply:
Ah cool. That's a neat idea. How do you run the command? I wonder if you could alias it.

Reply:
yea probably future iterations i make to this project will involve me aliasing it. this is my first time working on a repo that is a node package so i just went with it.

Reply:
This feels weird. The script lives in `bin/blendid.js`, so if you need to run `blendid` within the repo, just run that script.

Reply:
ah good to know. yea i will remove this package.